### DISCUSSION TITLE
How to show find_package usage for a installed package

### DISCUSSION THREAD

**Discussion by huhuang03** [microsoft/vcpkg#34283] [Q&A] [answered]
I installed an package called foo.

When I first time install it, will print usage info:

find_package(foo)
target_link_library(main foo:foo)

But some day passed, I forget how to use this foo package. Can I print out this usage message again?

**Comment by Osyotr** [comment, marked as answer]
Calling `vcpkg install <package>` will print usage if the package is already installed.

**Reply by huhuang03** [reply]
Yes, it works. Although I remember it didn’t seem like this in earlier versions.

**Comment by dg0yt** [comment]
https://vcpkg.link/ shows usage for all packages.

**Reply by Osyotr** [reply]
It doesn't show autogenerated usage though.

**Reply by huhuang03** [reply]
Yes, this site can also show the usage.
![image](https://github.com/microsoft/vcpkg/assets/4716706/2aa65ba1-1588-4569-9bcb-3294d5806fd1)

**Reply by ChivenZhang** [reply]
https://vcpkg.link/ports/fastlz
That's too bad for me, there is no usage for fastlz.
